### PR TITLE
Tests fix

### PR DESCRIPTION
**Pull request by deniszh**
- pining ubuntu
 - removing go 1.13

### CODE REVIEW

File: carbonserver/carbonserver.go [hunk 1]
```diff
@@ -837,7 +837,7 @@ func (listener *CarbonserverListener) updateFileList(dir string, cacheMetricName
 
 	var freeSpace uint64
 	// diskspace can be negative and Bavail is therefore int64
-	if stat.Bavail >= 0 { // nolint:staticcheck skipcq: SCC-SA4003
+	if stat.Bavail >= 0 { // nolint:staticcheck // skipcq: SCC-SA4003
```
Comment (bom-d-van) [inline carbonserver/carbonserver.go:840]: Nice catch!

Comment (deniszh) [reply carbonserver/carbonserver.go:840]: That's not me, but golangci-lint 1.38
But still dunno why "TestThrottleChan" failing, it's working on my laptop :D

Comment (bom-d-van) [reply carbonserver/carbonserver.go:840]: The test seems to be a bit unstable. Sometimes it passed,  sometimes it failed.
I think we can try modify the 0.95 to 0.9.

Comment (bom-d-van) [reply carbonserver/carbonserver.go:840]: Or I could re-phase it as the current newThrottleTicker can't meet the [0.95, 1.05] range 100% of the time. :P

Comment (deniszh) [reply carbonserver/carbonserver.go:840]: Strange, for me it's  failing 100% on GHA and passing 100% on my laptop. Trying 90%-110% range now.

Comment (deniszh) [reply carbonserver/carbonserver.go:840]: Ok, working now, merging

**Pull request merged**